dist/tokens/native/themes/primitives.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Do not edit directly
+ * Generated on Wed, 31 Jan 2024 14:36:33 GMT
+ */
+
+module.exports = {theme: "primitives",
+boxShadows: {},
+opacity: {},
+typography: { base: {  },base: {  },base: { 0: "n" ,1: "o" ,2: "n" ,3: "e"  },base: { 0: "n" ,1: "o" ,2: "n" ,3: "e"  },desktop: {  },desktop: {  },desktop: {  },desktop: {  },desktop: {  },desktop: {  },desktop: {  },desktop: {  },mobile: {  },mobile: {  },mobile: {  },xlarge: {  },large: {  },normal: {  },small: {  },sansSerif: { 0: "M" ,1: "u" ,2: "l" ,3: "i" ,4: "s" ,5: "h"  },serif: { 0: "B" ,1: "e" ,2: "s" ,3: "l" ,4: "e" ,5: "y"  },semiBold: { 0: "7" ,1: "0" ,2: "0"  },base: { 0: "5" ,1: "0" ,2: "0"  },light: { 0: "4" ,1: "0" ,2: "0"  },strong: { 0: "6" ,1: "0" ,2: "0"  },bold: { 0: "9" ,1: "0" ,2: "0"  }  },};
```

Comment:
Do we need this file?

Reply:
No, I'm not sure why this is being generated... 🤔

Reply:
Figured it out! It was because I had added in `token.attributes.category === "typography"` to the `themeTokens` filter, and because `primitives` has typography type of values, it was generating an associated theme file